src/CMakeLists.txt

```diff
@@ -101,7 +111,7 @@ set ( CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/../LICENSE.txt" )
 install( FILES ${CPACK_RESOURCE_FILE_LICENSE} DESTINATION share/doc/atmi COMPONENT runtime )
```

Comment:
Will the license file path still be accurate with the reorg?

Reply:
For  File Reorg , the license file path should be share/doc/component-name .
The license path is as expected for ATMI